libs/@blockprotocol/design-system/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@blockprotocol/design-system",
+  "version": "0.0.1",
+  "description": "Implementation of the Block Protocol Graph service specification for blocks and embedding applications",
```

Comment:
We should update this description to reflect the fact that this is the block design system, and not an implementation of the graph service.

Reply:
@nonparibus, Done